### PR TITLE
ROTATE_FLIP_MIRROR

### PR DESCRIPTION
**Pull request by Pooja-Mahadev-Soundalgekar**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -736,35 +742,69 @@
...
                                 if (cellvar.model.evtout.height == null) {
                                     commandPort = 0;
-								}
+                                }
                                 else {
                                     commandPort = cellvar.model.evtout.height;
-								}
+                                }
+                                var flip = 'false', mirror = 'false';
+                                function Styling(style)
```
Comment (kamakshidasan) [inline index.html]: These variables should be of datatype boolean and not string!

Comment (kamakshidasan) [inline index.html]: Does not conform to JavaScript naming conventions!

File: rotate.js [hunk 2]
```diff
@@ -0,0 +1,503 @@
+/*
+    @pooja,
+    rotate.js contains the implementation of
```
Comment (kamakshidasan) [inline rotate.js]: Is the name of this file justified?

**Comment by kamakshidasan** [comment]
Shall I close this request?

**Comment by Pooja-Mahadev-Soundalgekar** [comment]
Yes

**Pull request closed without merging**